src/librustc_expand/mbe/macro_rules.rs

```diff
@@ -181,6 +181,19 @@ impl TTMacroExpander for MacroRulesMacroExpander {
     }
 }
 
+struct MacroRulesDummyExpander;
```

Comment:
You don't need a struct and an impl for this, a free function is enough, see `SyntaxExtension::dummy_bang` which also does this.